components/Layout.js

```diff
@@ -1,32 +1,44 @@
 import Link from 'next/link'
 import { useState, useEffect } from 'react'
 import { getUsername } from '../lib/userAuth'
+import { getUserId } from '../lib/userAuth'
 
-const Layout = ({ children }) => {
-  const [ username, setUsername ] = useState('')
+const Layout = ({ children, home }) => {
```

Comment:
You can reply to this comment and tell me what the purpose of the `home` variable is.